services/user.py

```diff
@@ -0,0 +1,62 @@
+from django.contrib.auth import get_user_model
+from django.shortcuts import get_object_or_404
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return get_object_or_404(get_user_model(), pk=user_id)
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = get_user(user_id)
```

Comment:
The `update_user` function should handle the case where a user with the given ID does not exist. Consider using `get_object_or_404` or handling the `User.DoesNotExist` exception to provide a more user-friendly error message.